roles/contiv_network/tasks/ovs.yml

```diff
@@ -53,9 +53,16 @@
     - "tcp:127.0.0.1:6640"
     - "ptcp:6640"
 
+- name: check selinux status
+  shell: sestatus | egrep "SELinux status:.*enabled"
+  register: selinux_set
+  when: (ansible_os_family == "RedHat")
```

Comment:
following fix will be needed in my approach above, if we go with that:

`match("SELinux status:.*enabled")` => `search(".*SELinux status:.*enabled.*")`